flow/core/kernel/vehicle/traci.py

```diff
@@ -964,6 +967,20 @@ def apply_acceleration(self, veh_ids, acc):
                 next_vel = max([this_vel + acc[i] * self.sim_step, 0])
                 self.kernel_api.vehicle.slowDown(vid, next_vel, 1e-3)
 
+    def apply_acceleration_not_smooth(self, veh_ids, acc):
+        """See parent class."""
+        # to hand the case of a single vehicle
+        if type(veh_ids) == str:
+            veh_ids = [veh_ids]
+            acc = [acc]
+
+        for i, vid in enumerate(veh_ids):
+            if acc[i] is not None and vid in self.get_ids():
+                self.__vehicles[vid]["accel"] = acc[i]
+                this_vel = self.get_speed(vid)
+                next_vel = max([this_vel + acc[i] * self.sim_step, 0])
+                self.kernel_api.vehicle.setSpeed(vid, next_vel)
```

Comment:
how is this different than the normal method?

Reply:
the normal method uses slowDown() method which applies some sort of smoothing. here setSpeed() method is used which doesn't apply smoothing.